modules/api/src/main/java/org/eclipse/xpanse/api/response/Response.java

```diff
@@ -26,7 +29,7 @@ public class Response {
     @Schema(description = "The success boolean of response.")
     private Boolean success;
     @Schema(description = "The result data of response.")
-    private Object data;
+    private T data;
```

Comment:
Looks like we lost the important changes.

Reply:
I reset the commit and push force. Now, it is ok.